src/actions/getBookmark.js

```diff
@@ -0,0 +1,50 @@
+import request from '../utils/request';
+import actionTypeGenerator from './typeGenerator';
+
+export const getBookmarkType = actionTypeGenerator('GET_BOOKMARK');
+
+/**
+ * Action generator that is dispatched when operation is successful
+ * @param {string} message The response message
+ * @returns {object} The action to dispatch
+ */
+export const getBookmarkAction = (type, payload) => ({
+  type,
+  payload,
+});
+
+/**
+ * reset password action creator
```

Comment:
**Nitpick**

- This is not an action for reset password is it?

Reply:
resolved @devdbrandy